### ISSUE TITLE
Issue regarding booktree on unRAID

### ISSUE THREAD

**Issue by Dental1347** [myxdvz/booktree#1]
Hello!

I went ahead and used the following command to execute a dry run for my library:
```
python3 booktree.py mam-audible --source_path /mnt/user/data/torrents/readar
r --media_path /mnt/user/data/media/audiobooks --dry-run
  File "/mnt/user/appdata/scripts/booktree/booktree.py", line 55
    print (f"{str(row["book"])} is new, adding the book")
                       ^
SyntaxError: f-string: unmatched '['
```
I also tried the command with python (instead of python3),
```
root@Hypnos:/mnt/user/appdata/scripts/booktree# python booktree.py mam-audible --source_path /mnt/user/data/torrents/readarr --media_path /mnt/user/data/media/audiobooks --dry-run
  File "booktree.py", line 55
    print (f"{str(row["book"])} is new, adding the book")
                      ^
SyntaxError: invalid syntax
```

Thanks in advance, please let me know if I did something wrong.

**Comment by Dental1347** [comment]
For added context, I have also made sure to set this up in a venv:
```
#!/bin/bash
cd /mnt/user/appdata/scripts/booktree
source requirements/bin/activate
python booktree.py mam-audible --source_path /mnt/user/data/torrents/readarr --media_path /mnt/user/data/media/audiobooks --dry-run
cd /mnt/user/appdata/scripts/booktree
deactivate
```
Unfortunately, the same result.

**Comment by myxdvz** [comment]
I don't think this is the issue, because you are in the very beginning of the code... since you're not passing a session ID, you should just use the audible mode

python booktree.py audible --source_path /mnt/user/data/torrents/readarr --media_path /mnt/user/data/media/audiobooks --dry-run

**Comment by myxdvz** [comment]
i can confirm that this issue is happening on python 3.9.  The dependency mentioned in the read me is >= 3.10.  The code is using functions not available in 3.9
